packages/reporters/cli/src/SimpleCLIReporter.js

```diff
@@ -88,11 +88,11 @@ function writeOut(message: string): void {
   stdout.write(message + '\n');
 }
 
-function writeErr(message: string | Error): void {
+function writeErr(message: string | Error, level: LogLevel): void {
   let error = prettyError(message, {color: false});
   // prefix with parcel: to clarify the source of errors
   writeErrLine('parcel: ' + error.message);
-  if (error.stack != null) {
+  if (error.stack != null && logLevels[level] >= logLevels.verbose) {
```

Comment:
Maybe we should always log stacktraces in the SimpleCLIReporter since it's more utilitarian, and suppress them in non-verbose modes only in the main, user-friendly cli?